datalad/distribution/tests/test_clone.py

```diff
@@ -362,6 +363,9 @@ def test_clone_report_permission_issue(tdir):
         )
 
 
+# Started to hang on appveyor, only in the first run
```

Comment:
As I just saw you note in another PR, this isn't specific to a particular run.

Reply:
> As I just saw you note in another PR

Never mind, [that][0] is about the codecov failure, so please ignore this bit of my statement.

[0]: https://github.com/datalad/datalad/pull/3931#issuecomment-566147454

Reply:
Hmm, I see you marked this as resolved, but you didn't say how it was (and I don't see an additional push).  Unless I'm misreading your comment, I provided several examples that showed it was inaccurate.

Reply:
I thought 'nevermind' was the resolution

Reply:
I quoted a specific piece of my previous commented, and said "ignore this bit of my statement".

Reply:
ok, then I misunderstood you.  Please to push a fix to the comment

Reply:
Done as part of gh-3949.